charts/stellar-disbursement-platform/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: stellar-disbursement-platform
 description: A Helm chart for the Stellar Disbursement Platform Backend (A.K.A. `sdp`)
-version: 2.0.0
+version: 2.0.1
 appVersion: "2.0.0"
```

Comment:
🤔 should we keep the helm version in sync with the app version?

Reply:
I don't think they need to remain in sync. This is an example where the Chart changes without involving an application change.